modules/nw-externalip-about.adoc

```diff
@@ -0,0 +1,178 @@
+====
+
+You can assign an external IP address in the following ways:
+
+Automatic assignment of an external IP::
+{product-title} automatically assigns an IP address from the `autoAssignCIDRs` CIDR block to the `spec.externalIPs[]` array when you create a Service CR object.
+In this case, {product-title} implements a non-cloud version of the load balancer Service type and assigns IP addresses to the services.
+
+Manual assignment of an external IP::
+{product-title} uses the IP addresses assigned to the `spec.externalIPs[]` array when you create a Service CR. You cannot specify an IP address that you have already used in another Service.
+
+[id="configuration-externalip_{context}"]
+== Configuration for ExternalIP
+
+Use of an external IP address in {product-title} is governed by the following fields:
+
+* `spec.externalIP.autoAssignCIDRs` defines an IP address block used by the load balancer when choosing an external IP address for the service. {product-title} supports only a single IP address block for automatic assignment. This can be simpler than having to manage the port space of a limited number of shared IP addresses when manually assigning ExternalIPs to services. If automatic assignment is enabled, a Service with `spec.type=LoadBalancer` is allocated an external IP address.
+* `spec.externalIP.policy` defines the permissible IP address blocks when manually specifying an IP address. {product-title} does not apply policy rules to IP address blocks defined by `spec.externalIP.autoAssignCIDRs`.
+
+If routed correctly, external traffic from the configured external IP address block can reach service endpoints through any TCP or UDP port that the service exposes.
+
+[CAUTION]
+====
+You must ensure that the IP address block you assign terminates at one or more nodes in your cluster.
+====
+
+{product-title} supports both the automatic and manual assignment of IP
+addresses, and each address is guaranteed to be assigned to a maximum of one
+service. This ensures that each service can expose its chosen ports regardless
+of the ports exposed by other services.
+
+[NOTE]
+====
+To use IP address blocks defined by `autoAssignCIDRs` in {product-title}, you must configure the necessary IP address assignment and routing for your host network.
+====
+
+The following YAML describes a Service CR with an external IP configured:
+
+.Example Service object with `spec.externalIPs[]` set
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: http-service
+spec:
+  clusterIP: 172.30.163.110
+  externalIPs:
+  - 192.168.132.253
+  externalTrafficPolicy: Cluster
+  ports:
+  - name: highport
+    nodePort: 31903
+    port: 30102
+    protocol: TCP
+    targetPort: 30102
+  selector:
+    app: web
+  sessionAffinity: None
+  type: LoadBalancer
+status:
+  loadBalancer:
+    ingress:
+    - ip: 192.168.132.253
+----
+
+[id="restrictions-on-ip-assignment_{context}"]
+== Restrictions on IP address assignment
```

Comment:
Probably worth noting that this is external IP address assignment, unless it's clear from the context.

Reply:
Done!